roles/ceph-nfs/tasks/start_nfs.yml

```diff
@@ -1,4 +1,19 @@
 ---
+- name: check if rados index object exists
+  shell: "rados -p {{ cephfs_data }} ls|grep ganesha_rados_export_index"
```

Comment:
Are you hard coding the name of the object used as an index of RADOS export object URLS? It needs to be a configurable as in the manila driver, right?

Reply:
Not necessarily (this value is currently hardcoded in templates) but you are right it would be better to make this configurable. I'll update it in this PR.